### PR TITLE
link to spring5 generator

### PR DESCRIPTION
**Pull request by jessitron**
fixes 207

### CODE REVIEW

File: docs/developer/client.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Before you can build and run your own automation client,
 
 An automation client project is any project that connects to the
 Atomist development automation API.  The reference implementation of
-the automation client is the [automation-client-ts][client-ts]
+the automation client is the [automation-seed-ts][seed]
```
Comment (ddgenome) [inline docs/developer/client.md]: This change is not correct. The reference implementation of an automation client is automation-client-ts. automation-client-ts is what you would want to use as a seed when creating your own automation client project using a generator.

Comment (ddgenome) [reply docs/developer/client.md]: The following paragraph makes this distinction.

Comment (jessitron) [reply docs/developer/client.md]: what? that's the library. we talked about this in !zoom

Comment (jessitron) [reply docs/developer/client.md]: you want to use that as a library, _not_ as a seed. Never clone client-ts

Comment (jessitron) [reply docs/developer/client.md]: maybe "reference implementation" means something different to us? yeah, it must. maybe I'm just confused about the meaning of that phrase. Maybe other people won't be.

**Comment by jessitron** [comment]
@ddgenome how about this change?

**Pull request merged**